### ISSUE TITLE
Release a version with updated Angular dependency later than 1.2.x

### ISSUE THREAD

**Issue by jesperronn** [angular-google-chart/angular-google-chart#191]
Since the last release of this project 0.0.11, Angular has been released in both 1.3 and 1.4 serires.

Package.json defines `"angular": "~1.2.9"`, which resolves to latest 1.2 series 1.2.28

https://github.com/angular-google-chart/angular-google-chart/blob/0.0.11/package.json#L19

I would like if you can release a version accepting 1.4 and 1.3 series as well. I am currently running with angular 1.4.1 (newest) and have no problems.

Any feedback appreciated from others who run angular-google-chart with Angular 1.4 or 1.3 series?

**Comment by jesperronn** [comment]
In case the dependency to angular has to be broader but not open in the top, there are some examples in the [`package.json` documentation](https://docs.npmjs.com/files/package.json#dependencies):

* version1 - version2 Same as >=version1 <=version2.
* range1 || range2 Passes if either range1 or range2 are satisfied.

examples:

```
{ "dependencies" :
  { "foo" : "1.0.0 - 2.9999.9999"
  , "bar" : ">=1.0.2 <2.1.2"
  , "baz" : ">1.0.2 <=2.3.4"
  , "qux" : "<1.0.0 || >=2.3.1 <2.4.5 || >=2.5.2 <3.0.0"
```

This way, we could define the dependency as working with 1.2-1.4

**Comment by Balrog30** [comment]
I run the latest 1.3.x with no problems.  Ultimately, I'd like to add some tests to the build process.  It should be possible to run that tests in the latest patch of each minor version, just to be sure.  I've yet to test in 1.4, but we aren't really using any parts of angular that are moving targets.  I was looking at 1.2 and 1.3 angular docs while refactoring the project for extensibility, and there was nothing significantly different in any of the features of Angular that the project touches.

Unless there's some radical shift in how directives function (a change I think they are not likely to make in 1.x, with 2.0 on the way), then I think it is safe to assume open-ended compatibility with 1.x.  The depth of the Angular code in this project is actually pretty shallow.  Most of the complex stuff is handled by the Google Charts API in plain JavaScript.

**Comment by Balrog30** [comment]
Version 0.1.0-beta.2 has been released, with `package.json` and `bower.json` both having angular dependency declared as `>1.2.8`. This whole dependency version thing has probably been the single largest issue to constantly come up over and over again, so I'm certain that this doesn't put the issue to rest entirely, but at least people can stop complaining that it requires them to manually resolve the version conflict.

That said, I've started to write unit tests for some existing, and some new code, and I plan to set up the unit tests to run against the latest of each minor version of angular. For example, (1.2.x, 1.3.x, 1.4.x, 1.5.x, etc.) until Angular 2, in which case from what I'm hearing there might still be a call to use a component to use Google Charts for convenience, but the component system could look entirely different.

Anyway, I think this issue can be put to rest for today. This beta release changes the package meta data, and as long as I don't get any reports of insane bugs, I think we're on track to roll this out as an official release by the end of the month.
